File: Studiopro/src/app/models/Common/SystemMetadatum.ts
interface SystemMetadatum {
    MetadataId: number;

    Category: string;

    Code: string;

    DisplayName: string;

    NumericValue: number | null;

    SortOrder: number | null;

    IsActive: boolean | null;

    IsSystem: boolean;

    Description: string | null;

    CreatedDate: Date | string | null;

    ModifiedDate: Date | string | null;
}

File: Service/Models/SystemMetadatum.cs
using System;
using System.Collections.Generic;

namespace Service.Models;

public partial class SystemMetadatum
{
    public int MetadataId { get; set; }

    public string Category { get; set; } = null!;

    public string Code { get; set; } = null!;

    public string DisplayName { get; set; } = null!;

    public int? NumericValue { get; set; }

    public int? SortOrder { get; set; }

    public bool? IsActive { get; set; }

    public bool IsSystem { get; set; }

    public string? Description { get; set; }

    public DateTime? CreatedDate { get; set; }

    public DateTime? ModifiedDate { get; set; }
}
